polkadot/runtime/rococo/src/lib.rs

```diff
@@ -1504,6 +1508,8 @@ construct_runtime! {
 
 		// Sudo.
 		Sudo: pallet_sudo::{Pallet, Call, Storage, Event<T>, Config<T>} = 255,
+
+		RootTesting: pallet_root_testing::{Pallet, Call, Storage, Event<T>} = 249,
```

Comment:
I added random numbers here and in `Westend`'s runtime. Is it ok?

Reply:
Please put in front of the Sudo, so that the order of the pallets and their indices line up.

Reply:
Do you mean something like this?

```rust
                // Root testing pallet.
		RootTesting: pallet_root_testing::{Pallet, Call, Storage, Event<T>} = 249,

		// Sudo.
		Sudo: pallet_sudo::{Pallet, Call, Storage, Event<T>, Config<T>} = 255,
```

Reply:
Yea 254 is already taken, guess its okay.